downloads/swan-lake-release-notes/swan-lake-2201.4.0/RELEASE_NOTE.md

```diff
@@ -87,6 +87,27 @@ To view bug fixes, see the [GitHub milestone for 2201.4.0 (Swan Lake)](https://g
 
 ### Improvements
 
+#### CLI
+##### Refactored the `bal format` command
```

Comment:
@praneesha Shouldn't we ideally have the component names under H4 headers? (e.g `Formatter`, `JSON-to-record Converter`, etc)?